src/agent/communicator/src/http_client.cpp

```diff
@@ -10,6 +10,8 @@
 
 namespace http_client
 {
+    constexpr int CONNECTION_RETRY_MSECS = 5000;
```

Comment:
This shouldn't exist, now we have the configurable setting.

Reply:
That's what I was asking earlier. Wasn't sure whether to use the same setting. Done.